service/api/conversions.go

```diff
@@ -261,7 +261,7 @@ func VariableValueAsString(v *proc.Variable) string {
 		return convertFloatValue(v, 32)
 	case reflect.Float64:
 		return convertFloatValue(v, 64)
-	case reflect.String, reflect.Func:
+	case reflect.String, reflect.Func, reflect.Struct:
```

Comment:
All structs?

Reply:
I don't think there's a reason to be more specific.